diff --git a/frontend/prizm/index.html b/frontend/prizm/index.html
index efb3be6..d9e32c6 100644
--- a/frontend/prizm/index.html
+++ b/frontend/prizm/index.html
@@ -16,7 +16,7 @@
   -->
   </head>
 
-  <body class="bg-light" id="page-top">
+  <body id="page-top">
     <div id="app"></div>
 
     <hr class="mt-4 mb-4">
diff --git a/frontend/prizm/src/App.vue b/frontend/prizm/src/App.vue
index 1e49ad2..eb99b86 100644
--- a/frontend/prizm/src/App.vue
+++ b/frontend/prizm/src/App.vue
@@ -1,6 +1,6 @@
 <template>
 <div>
-  <nav class="navbar navbar-expand-lg navbar-light sticky-top text-uppercase" id="mainNav" style="background-color: white; border-bottom: 1px solid rgba(0,0,0,0.1)">
+  <nav class="navbar navbar-expand-lg navbar-light bg-light sticky-top text-uppercase" id="mainNav">
     <router-link class="navbar-brand" to="/">Refair.Me</router-link>
     <button class="navbar-toggler navbar-toggler-right text-uppercase bg-primary text-white rounded" type="button" data-toggle="collapse" data-target="#navbarResponsive" aria-controls="navbarResponsive" aria-expanded="false" aria-label="Toggle navigation">
       <span class="navbar-toggler-icon"></span>
@@ -94,6 +94,3 @@ export default {
   }
 }
 </script>
-<style lang="sass">
-
-</style>
diff --git a/frontend/prizm/src/components/job-add.vue b/frontend/prizm/src/components/job-add.vue
index 8a5d3ed..2262e7a 100644
--- a/frontend/prizm/src/components/job-add.vue
+++ b/frontend/prizm/src/components/job-add.vue
@@ -1,6 +1,4 @@
 <template>
-  <div class="card">
-    <div class="card-body">
 <form>
   <h5>TELL US ABOUT YOUR JOB</h5>
   <hr>
@@ -70,8 +68,6 @@
   </div>
   <button type="button" class="btn btn-success" v-on:click="addJob" :disabled="!isAuthenticated"> Add new job </button>
 </form>
-</div>
-</div>
 </template>
 <script>
 import store from '../../prizm-vuex.js'
diff --git a/frontend/prizm/src/components/job-details.vue b/frontend/prizm/src/components/job-details.vue
index 15e6363..c65bad7 100644
--- a/frontend/prizm/src/components/job-details.vue
+++ b/frontend/prizm/src/components/job-details.vue
@@ -1,32 +1,28 @@
 <template>
 <div>
-  <div class="card">
-    <div class="card-body">
-      <h1>
-        {{job.jobtitle}}
-        <button class="btn btn-success float-right col-3"
-                @click="apply()">
-          Apply
-        </button>
-      </h1>
-      <h4 class="text-muted">{{job.required_fund}} PLN</h4>
-      <small class="badge badge-pill badge-light">REMOTE = {{job.required_remote}}</small>
-      <small class="badge badge-pill badge-light">RELOCATION = {{job.required_relocation}}</small>
+  <h1>{{job.jobtitle}}</h1>
+  <div class="row">
+    <div class="col-9">
+      <p>{{job.description}}</p>
     </div>
-  </div>
-  <div class="card mt-3">
-    <div class="card-body">
-      <h4>Description</h4>
-      <div class="row">
-        <div class="col-6 text-muted">
-          {{job.description}}
-        </div>
-        <div class="col-6">
-          <lineChartKw style="width: 100%" chart-id="kwchart" :height="250" :chart-data="patterndatakw"></lineChartKw>
-        </div>
-      </div>
+    <div class="col-3" style="border-left: 1px solid black">
+      <button class="btn btn-success" style="width: 100%" @click="apply()">Apply</button>
+      <p>Remote:
+        <span v-if="job.required_remote == 1">required</span>
+        <span v-else>not required</span>
+      </p>
+      <p>Relocation:
+        <span v-if="job.required_relocation == 1">required</span>
+        <span v-else>not required</span>
+      </p>
+      <p>Salary: <span>{{job.required_fund[0]}} - {{job.required_fund[1]}}</span></p>
+      <p>Essentials: <span>{{job.essentials}}</span></p>
+
     </div>
   </div>
+  <div class="row" >
+    <lineChartKw style="width: 100%" chart-id="kwchart" :height="200" :chart-data="patterndatakw"></lineChartKw>
+  </div>
 </div>
 </template>
 <script>
@@ -44,7 +40,7 @@ Vue.component('lineChartKw', {
   mixins: [mixins.reactiveProp],
   props: ['chartData'],
   mounted() {
-    if (this.chartData) {
+    if(this.chartData) {
       this.renderChart(
         this.chartData, {
           responsive: true,
@@ -70,7 +66,7 @@ export default {
       console.log(ret.data.weights)
       this.$data.patterndatakw = {
         labels: ['Backend', 'Full Stack', 'Mobile/Embedded', 'Testing', 'Frontend', 'Dev Ops', 'Business Intelligence', 'IT Trainee', 'Project Management', 'Support', 'UX Designer', 'Business Analyst', 'Other'],
-        datasets: [{
+        datasets:[{
           label: 'Refair.me Profile',
           backgroundColor: '#a84979',
           data: ret.data.weights
@@ -81,11 +77,8 @@ export default {
   data() {
     return {
       job: {
-        jobtitle: 'Title',
-        description: 'Desc',
-        required_remote: true,
-        required_relocation: true,
-        required_fund: '10000'
+        jobtitle: '',
+        description: ''
       },
       patterndatakw: {
         labels: ['Backend', 'Full Stack', 'Mobile/Embedded', 'Testing', 'Frontend', 'Dev Ops', 'Business Intelligence', 'IT Trainee', 'Project Management', 'Support', 'UX Designer', 'Business Analyst', 'Other'],
@@ -98,7 +91,7 @@ export default {
     }
   },
   methods: {
-    apply: function() {
+    apply: function () {
       var id = this.$route.params.id;
       console.log("apply: " + id + " " + this.$store.state.dehashedData.EMAIL)
 
@@ -113,7 +106,7 @@ export default {
         console.log(ret)
       })
     },
-    matchJobs: function() {
+    matchJobs: function () {
       var id = this.$route.params.id;
       console.log("match jobs")
       const self = this;
@@ -132,7 +125,4 @@ export default {
     color: gray
     span
       color: black
-  .card
-    box-shadow: 0 2px 6px 0 hsla(0,0%,0%,0.1)
-    border: 0
 </style>
diff --git a/frontend/prizm/src/components/keywords.vue b/frontend/prizm/src/components/keywords.vue
index 8627a1f..841796a 100644
--- a/frontend/prizm/src/components/keywords.vue
+++ b/frontend/prizm/src/components/keywords.vue
@@ -1,22 +1,25 @@
 <template>
 <div>
-  <div>
-    <div class="input-group">
-      <input v-model="newKeyword" type="text" @keyup.enter="addKeyword" placeholder="name a keyword" class="bg-light">
-      <div class="input-group-append">
-        <button class="btn btn-outline-secondary" @click="addKeyword" type="button">Add</button>
-      </div>
+  <div class="card">
+    <div class="card-body">
+      <label @click="addKeyword">
+        Add keyword
+      </label>
+      <input v-model="newKeyword" type="text" @keyup.enter="addKeyword" placeholder="name a keyword">
+      </input>
+      <br/>
+      <small>
+        hint: use commas to add multiple keywords in one go<br>
+      </small>
+      <small>
+        <a href="#" title="GitHub">
+          or add GitHub profile
+        </a>
+      </small>
     </div>
-    <small>
-      hint: use commas to add multiple keywords in one go<br>
-    </small>
-    <small>
-      <a href="#" title="GitHub">
-        or add GitHub profile
-      </a>
-    </small>
   </div>
-  <span class="badge badge-pill badge-light text-muted" v-for="keyword in akeywords">
+
+  <span class="badge badge-pill badge-light" v-for="keyword in akeywords">
     <label>
       <input type="checkbox"
              name="optinkeyword"
@@ -79,16 +82,9 @@ export default {
 </script>
 
 <style lang="sass" scoped>
-  .badge
-    font-size: 1.1em
-    cursor: pointer
-    font-weight: normal
-    margin: 3px
-  input
-    border: 1px solid rgba(0,0,0,0.2)
-    border-radius: 3px
-    padding-left: 10px
-  .input-group-append
-    button
-      border-color: rgba(0,0,0,0.2)
+.badge
+  font-size: 1.1em
+  cursor: pointer
+  font-weight: normal
+  margin: 3px
 </style>
diff --git a/frontend/prizm/src/components/profilebuild.vue b/frontend/prizm/src/components/profilebuild.vue
index 4d66333..24a00ae 100644
--- a/frontend/prizm/src/components/profilebuild.vue
+++ b/frontend/prizm/src/components/profilebuild.vue
@@ -1,55 +1,45 @@
 <template>
 <div>
-
-  <div class="card">
-    <div class="card-body">
-      <div class="col-xs-12">
-        <h1 class="card-title" style="color: #B0AFAB;">Your Profile Builder</h1>
-      </div>
-      <div class="row">
-        <div class="col-xs-12 col-md-6">
-          <refairkeywords id="profilekeywords" v-on:keywords="updateProfileKeywords"></refairkeywords>
-        </div>
-        <div class="col-xs-12 col-md-6">
-          <lineChartKw chart-id="kwchart" :height="250" :chart-data="patterndatakw"></lineChartKw>
-        </div>
-      </div>
-
-      <div class="col-xs-12">
-        <a href="#" class="btn btn-info" @click="saveProfile">Save Profile</a>
-      </div>
+  <div class="col-xs-12">
+    <h1 style="color: #B0AFAB; font-size:4em;">Your Profile Builder</h1>
+  </div>
+  <div class="row">
+    <div class="col-xs-12 col-md-6">
+      <refairkeywords id="profilekeywords" v-on:keywords="updateProfileKeywords"></refairkeywords>
+    </div>
+    <div class="col-xs-12 col-md-6">
+      <lineChartKw chart-id="kwchart" :chart-data="patterndatakw"></lineChartKw>
     </div>
   </div>
 
-  <div class="card mt-3">
-    <div class="card-body">
-      <h1 style="color: #B0AFAB;">Profiled Jobs <font-awesome-icon v-if="loading" :icon="cogIcon" spin/></h1>
-
-      <table class="table table-striped">
-        <thead>
-          <tr class="text-muted">
-            <th scope="col">Id</th>
-            <th scope="col">Title</th>
-            <th scope="col">Pay</th>
-            <th scope="col">Job Profile</th>
-          </tr>
-        </thead>
-        <tbody>
-          <tr v-if="jobs.length == 0">
-            <td colspan='4' style="text-align: center">No jobs matched</td>
-          </tr>
-          <tr v-for="job in jobs" @click="openJobDetails(job.id)">
-            <th scope="row">{{job.id}}</th>
-            <td>{{job.title}}Title</td>
-            <td>{{job.pay}}$100</td>
-            <td style="width: 50%">
-              <lineChartKw chart-id="" :height="250" :chart-data="populateWeights(job.weights)"></lineChartKw>
-            </td>
-          </tr>
-        </tbody>
-      </table>
-    </div>
+  <div class="col-xs-12 col-md-6 mb-3">
+    <a href="#" class="btn btn-info" @click="saveProfile">Save Profile</a>
   </div>
+  <h1 style="color: #B0AFAB; font-size:3em;">Profiled Jobs <font-awesome-icon v-if="loading" :icon="cogIcon" spin/></h1>
+
+  <table class="table table-striped">
+    <thead>
+      <tr>
+        <th scope="col">Id</th>
+        <th scope="col">Title</th>
+        <th scope="col">Pay</th>
+        <th scope="col">Job Profile</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr v-if="jobs.length == 0">
+        <td colspan='4' style="text-align: center">No jobs matched</td>
+      </tr>
+      <tr v-for="job in jobs" @click="openJobDetails(job.id)">
+        <th scope="row">{{job.id}}</th>
+        <td>{{job.title}}</td>
+        <td>{{job.pay}}</td>
+        <td style="width: 50%">
+          <lineChartKw chart-id="" :height="200" :chart-data="populateWeights(job.weights)"></lineChartKw>
+        </td>
+      </tr>
+    </tbody>
+  </table>
 </div>
 </template>
 
@@ -197,12 +187,9 @@ export default {
   }
 }
 </script>
-<style lang="sass">
+<style lang="sass" scoped>
   .h1
     margin-bottom: 100px
   tbody tr
     cursor: pointer
-  .card
-    box-shadow: 0 2px 6px 0 hsla(0,0%,0%,0.1)
-    border: 0
 </style>
diff --git a/frontend/prizm/src/components/refair-new-job.vue b/frontend/prizm/src/components/refair-new-job.vue
index a7d9d53..e8ed1b3 100644
--- a/frontend/prizm/src/components/refair-new-job.vue
+++ b/frontend/prizm/src/components/refair-new-job.vue
@@ -1,7 +1,7 @@
 <template>
 <div>
   <div class="row">
-    <div class="col-12 col-md-4 mb-3" v-for="jobdesc in this.$data.refjobdescs">
+    <div class="col-12 col-md-4" v-for="jobdesc in this.$data.refjobdescs">
       <div class="card">
         <img class="card-img-top"
           src="https://placeimg.com/640/480/tech"
diff --git a/frontend/prizm/src/router/index.js b/frontend/prizm/src/router/index.js
index fcbb7c3..0ad021b 100644
--- a/frontend/prizm/src/router/index.js
+++ b/frontend/prizm/src/router/index.js
@@ -17,7 +17,6 @@ export default new Router({
       path: '/',
       name: 'home',
       component: home
-      //component: jobdetails
     },{
       path: '/auth/signup',
       name: 'signup',
